databricks-sdk-java/src/test/java/com/databricks/sdk/mixin/SecretsExtTest.java

```diff
@@ -0,0 +1,96 @@
+package com.databricks.sdk.mixin;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.databricks.sdk.service.workspace.*;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+public class SecretsExtTest {
+  @Mock SecretsService mockedSecretsService;
+
+  @BeforeEach
+  void setup() {
+    MockitoAnnotations.openMocks(this);
```

Comment:
I believe this returns an `Autocloseable` that needs to be closed after each test, if I'm not mistaken.